cluster/gce/config-test.sh

```diff
@@ -197,7 +197,7 @@ fi
 ENABLE_RESCHEDULER="${KUBE_ENABLE_RESCHEDULER:-true}"
 
 # If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely.
-ADMISSION_CONTROL="${KUBE_ADMISSION_CONTROL:-NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds,PodPreset}"
+ADMISSION_CONTROL="${KUBE_ADMISSION_CONTROL:-NamespaceLifecycle,LimitRanger,ServiceAccount,PersistentVolumeLabel,PodPreset,DefaultStorageClass,DefaultTolerationSeconds,ResourceQuota}"
```

Comment:
@jessfraz was there a reason PodPreset was added after ResourceQuota in https://github.com/kubernetes/kubernetes/commit/fe84363ce2ec067eea48fcc1d45ee8943d1eccbf#diff-e6e5eefc330d75c8ef2e218de62c5fdf ?

I'm assuming there wasn't, and moved it earlier in the chain, since resourcequota needs to act on the final state of the pod